docs/cli/argo_archive.md

```diff
@@ -6,6 +6,27 @@ manage the workflow archive
 argo archive [flags]
 ```
 
+### Examples
+
+```
+
+   		# List workflow archives:
+   		argo archive list
+   		# Get details of a specific workflow archive:
+   		argo archive get uid
+   		# Delete a specific workflow archive:
+   		argo archive delete uid
+   		# List workflow archives by label key:
+   		argo archive list-label-key
+   		# List workflow archives by label value:
+   		argo archive list-label-value
+   		# Resubmit a workflow archive:
+   		argo archive resubmit uid
+		# Retry a workflow archive:
+   		argo archive retry uid
```

Comment:
so this whole section is actually already repeated in the "See Also" section below. and in the sidebar in the docs as well. and in the CLI help output already.

so I don't think we should duplicate documentation here or in any `root` command really. 

if we have anything here, I suppose we could have:

```sh
# Print this message
argo archive

# Otherwise, see subcommands below
```